### ISSUE TITLE
How to add condition to shipping fee only? 

### ISSUE THREAD

**Issue by everisk** [darryldecode/laravelshoppingcart#315]
For example, we have a coupon discount delivery fee of xx amnt and instead of applying this discount to subtotal, we need to apply the discount only when shipping fee exists. Is there a method to add shipping fee and have condition apply to target "shipping" instead of just subtotal or total?

**Comment by onyxtec** [comment]
Yes it exists at least now. I believe it didn't exist in old versions but should be there in 4.1+ at least if not before.
